src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -283,15 +283,7 @@ public boolean isPlantable(final ItemStack itemStack)
             }
 
             String[] split = string.split(":");
-            //todo, this is backwards compatibility, remove in 1.16
```

Comment:
This clearly states remove in **1.16** why was this removed in a PR that targets _1.15_